cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -669,3 +673,67 @@ def test_import_report_preserve_owner(preserve_owner, create_po_user_and_group,
 
     assert view.report_info.user.text == user
     assert view.report_info.group.text == group
+    pass
+
+
+@pytest.fixture
+def setup_vm(configure_fleecing, appliance, provider):
+    vm = appliance.collections.infra_vms.instantiate(
+        name=random_vm_name(context="report", max_length=20),
+        provider=provider,
+        template_name="env-rhel7-20-percent-full-disk-tpl",
+    )
```

Comment:
Will it be fine? if we add **template_name** in cfme_data.yml file and fetch this template which can be used for other fixtures or tests too. Or can we use any existing template instead of this one?
What do you think?

Reply:
This template is specific to this test, it probably won't be used anywhere else so I directly used the name.